HW_9/Task1.py

```diff
@@ -0,0 +1,71 @@
+#Напишите следующие функции:
+#*Нахождение корней квадратного уравнения.
+#*Генерация csv файла с тремя случайными числами в каждой строке 100-1000 строк.
+#*Декоратор, запускающий функцию нахождения корней квадратного уравнения с каждой тройкой чисел из csv файла.
+#*Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
+
+import csv
+import datetime
+import json
+import math
+import os.path
+from random import randint as rnd
+from typing import Callable
+
+
+def deco_csv(function: Callable):
+    create_coefficients_csvfile()
+
+    def wrapper():
+        with open('coefficiens.csv', 'r', encoding='UTF-8') as file:
+            data = csv.reader(file, quoting=csv.QUOTE_NONNUMERIC)
+            for coefficient in data:
+                if coefficient and coefficient[0] != 0:
+                    function(*coefficient)
+
+    return wrapper
+
+def json_result(func: Callable):
+    result = {}
+    if os.path.exists('solutions.json'):
+        with open('solutions.json', 'r', encoding='UTF-8') as file:
+            result = json.load(file)
+    else:
+        with open('solutions.json', 'w', encoding='UTF-8') as file:
+            json.dump(result, file)
```

Comment:
Наверное, так будет правильнее
```suggestion
    if os.path.exists('solutions.json'):
        with open('solutions.json', 'r', encoding='UTF-8') as file:
            result = json.load(file)
    else:
        result = {}
```